setup.py

```diff
@@ -64,18 +77,19 @@ def read(*rnames):
         'Topic :: Text Processing',
         'Topic :: Utilities',
     ],
-    author='rst2pdf maintainers',
-    author_email='maintainers@rstpdf.org',
-    description='Convert reStructured Text to PDF via ReportLab.',
-    long_description=read('README.rst'),
-    long_description_content_type='text/x-rst',
-    license='MIT',
-    keywords='restructured convert rst pdf docutils pygments reportlab',
-    url='https://rst2pdf.org',
+    author="rst2pdf maintainers",
+    author_email="maintainers@rstpdf.org",
+    description="Convert reStructured Text to PDF via ReportLab.",
+    long_description=long_description,
+    long_description_content_type="text/x-rst",
+    license="MIT",
+    keywords="restructured convert rst pdf docutils pygments reportlab",
+    url="https://rst2pdf.org",
     project_urls={
         'Bug Reports': 'https://github.com/rst2pdf/rst2pdf/issues',
         'Source': 'https://github.com/rst2pdf/rst2pdf',
     },
-    download_url='https://github.com/rst2pdf/rst2pdf/releases',
+    download_url="https://github.com/rst2pdf/rst2pdf/releases",
     entry_points={'console_scripts': ['rst2pdf = rst2pdf.createpdf:main']},
+    test_suite='rst2pdf.tests.test_rst2pdf.test_suite',  # TODO: this needs to be updated
 )
```

Comment:
It looks live you've reverted a lot of the changes from c3c8f3dd6 here. Was that intentional?

Reply:
Nope - means I used the wrong command when fixing the conflict!